src/language-js/print/call-arguments.js

```diff
@@ -318,11 +317,40 @@ function isHopefullyShortCallArgument(node) {
   return isRegExpLiteral(node) || isSimpleCallArgument(node);
 }
 
-function isReactHookCallWithDepsArray(args) {
+/**
+ * Gets `"foo"` from `foo()`.
+ * Gets `"foo"` from `bar.foo()`
+ * Gets `"foo"` from `bar.baz.foo()`
+ * @param {any} callExpression
+ * @returns {(string | null)}
+ */
+function getCallName(callExpression) {
+  const { callee } = callExpression;
+  switch (callee.type) {
+    case "Identifier":
+      return callee.name;
+    case "MemberExpression": {
```

Comment:
you didn't test this.

Reply:
Thank you, fixed [bd14cbd](https://github.com/prettier/prettier/pull/14303/commits/bd14cbd6aa0fd4f6556c2ba7a932ed9233a404f4)